### PR TITLE
Add test coverage for `lib/run_action.js`

### PR DESCRIPTION
**Pull request by rmeritz**
`run_action` didn't have any unit tests but was already pretty throughly tested because almost all methods use it run make their API requests. I didn't add unit testing I just added the missing pieaces of coverage:
- Dealing with retries
- Dealing with the user header if requests are from the browser

### CODE REVIEW

File: test/find.test.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ describe('record retrival', function() {
     });
 
     afterAll(function() {
+        delete global.x;
```
Comment (jugglinmike) [inline test/find.test.js:19]: This should probably delete `global.window`

File: test/find.test.js [hunk 2]
```diff
@@ -41,4 +43,61 @@ describe('record retrival', function() {
...
+            .base('app123')
+            .table('Table')
+            .find(recordId)
+            .then(function(foundRecord) {
+                expect(foundRecord.id).toBe(recordId);
+                expect(foundRecord.get('Name')).toBe('Rebecca');
+            });
+    });
+
+    it('can find after a retry if rate limited', function(done) {
```
Comment (jugglinmike) [inline test/find.test.js:72]: Nice!

File: test/find.test.js (hunk 1 above)
Comment (rmeritz) [reply test/find.test.js]: Opps thanks fixed.

**Comment by rmeritz** [comment]
@kasrak - Once this PR gets merged I'm ready to open up a stacked PR for changing `requests` with `node-fetch`: https://github.com/bocoup/airtable.js/compare/run-action-tests-rmeritz...bocoup:node-fetch-rmeritz

**Comment by kasrak** [comment]
@rmeritz thanks! Looks like there's a conflict with `test/find.test.js`

**Comment by rmeritz** [comment]
@kasrak - I fixed the merge conflict.

**Pull request merged**